pkg/keyring/keyring_darwin.go

```diff
@@ -13,14 +13,17 @@ import (
 // ClearCredentials removes any existing items in the keychain,
 // it will ignore if not found errors
 func ClearCredentials(prefix string) error {
-	for _, k := range []string{username, password, bearer} {
-		item := keychain.NewItem()
-		item.SetSecClass(keychain.SecClassGenericPassword)
-		item.SetService(keyringService)
-		item.SetAccount(format(prefix, k))
-		err := keychain.DeleteItem(item)
-		if err != nil {
-			return errors.New("failed to delete credential from keychain")
+	for _, k := range []string{username, password, bearer, refreshToken} {
+		key := format(prefix, k)
+		if _, err := QueryKeychain(key); err == nil {
+			item := keychain.NewItem()
+			item.SetSecClass(keychain.SecClassGenericPassword)
+			item.SetService(keyringService)
+			item.SetAccount(key)
+			err := keychain.DeleteItem(item)
+			if err != nil {
+				return errors.New("failed to delete credential from keychain")
+			}
 		}
```

Comment:
@mandopaloooza can you checkout this PR on your macbook?

Reply:
```
// Configure and signin
$ go run main.go configure
? Enter the url for the controller API (example https://appgate.controller.com/admin) https://envy-10-97-154-2.devops:8443/admin
Configuration updated successfully

$ go run main.go configure signin --no-verify
? Username: admin
? Password: *****
Successfully signed in

// Configure and signin with another env
$ go run main.go configure 
? Enter the url for the controller API (example https://appgate.controller.com/admin) https://envy-10-97-174-2.devops:8443/admin
Configuration updated successfully

$ go run main.go configure signin --no-verify
? Username: admin
? Password: *****
Successfully signed in
```

<img width="1009" alt="Screen Shot 2022-09-06 at 2 25 00 PM" src="https://user-images.githubusercontent.com/7606627/188553438-f83dc10c-9f15-46c3-acef-726d2170e145.png">